app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Recipes</h1>
+<h2><%= @recipe.name %></h2>
```

Comment:
It may make the view more readable and help you write more robust tests if you add labels to these attributes. Instead of: `<h2><%= @recipe.name %></h2>`, consider

`<h2>Name: <%= @recipe.name %></h2>`
`<h3>Complexity: <%= @recipe.complexity %></h3>`

Reply:
See notes in your test to see why this helps with testing too